inference/core/workflows/entities/types.py

```diff
@@ -313,6 +313,24 @@ def __hash__(self) -> int:
     docs=DETECTION_KIND_DOCS,
 )
 
+DETECTION_KIND_DOCS = """
```

Comment:
could you explain why this additional kind is needed?

Reply:
each sv detection wont necessarily fall exactly into one of the current types i.e. missing class, class_id, etc so I created the batch of Detection kind as a catch all